internal/controller/config/config.go

```diff
@@ -57,7 +57,7 @@ type SSHCredential struct {
 
 type TLS struct {
 	InsecureSkipVerify bool              `json:"insecureSkipVerify"`
-	CACertSecret       SecretKeySelector `json:"CACertSecret"`
+	CACertSecret       SecretKeySelector `json:"caCertSecret"`
 	CertSecret         SecretKeySelector `json:"certSecret"`
 	KeySecret          SecretKeySelector `json:"keySecret"`
 }
```

Comment:
Sorry, I was mistaken about incorporating all of those fields. They seem to be related to the client authenticating itself to the server. How about we just include `InsecureSkipVerify` field?

type TLS struct {	type TLS struct {
	InsecureSkipVerify bool              `json:"insecureSkipVerify"`		InsecureSkipVerify bool              `json:"insecureSkipVerify"`
}

Reply:
Yeah addressed.